source/vir/src/triggers_auto.rs

```diff
@@ -514,34 +522,72 @@ fn trigger_score(ctxt: &Ctxt, trigger: &Trigger) -> Score {
 
 // Find the best trigger that covers all the trigger variables.
 // This is a variant of minimum-set-cover, which is NP-complete.
```

Comment:
Can you please update this docstring given the new functionality? (It also needs three slashes (`///`) to conform with the rustdoc convention.)

Reply:
Done